pkg/querier/tripperware/utils/utils.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import (
+	"sort"
+
+	"github.com/cortexproject/cortex/pkg/cortexpb"
+)
+
+// SliceSamples assumes given samples are sorted by timestamp in ascending order and
+// return a sub slice whose first element's is the smallest timestamp that is strictly
+// bigger than the given minTs. Empty slice is returned if minTs is bigger than all the
+// timestamps in samples.
```

Comment:
nit: I think it's always a good idea to explicit called out what happens if caller violates assumption. Like "If the given samples is not sorted, the behaviour is undefined". Of course it's debatable that this is just stating the obvious; hence the nit :)

Reply:
Updated